### PR TITLE
Update Sydney chain specification #153

### PR DESCRIPTION
**Pull request by obsessed-cake**
## Issue

https://github.com/ggxchain/planning/issues/153

### CODE REVIEW

File: node/src/command.rs [hunk 1]
```diff
@@ -60,6 +60,8 @@ impl SubstrateCli for Cli {
 			"" | "local" => Box::new(chain_spec::local_testnet_config()?),
 			// on """release""", replace with  included resource
 			#[cfg(not(feature = "brooklyn"))]
+			"sydney-devnet" => Box::new(chain_spec::sydney_devnet_config()?),
```
Comment (uchu) [inline node/src/command.rs:63]: @obsessed-cake What do you mean by "sydney-devnet"? As basically Sydney is always a public testnet. Just one chain.

**Review by uchu — COMMENTED** [review]
Jut a question regarding devnet->testnet naming.

Brooklyn is devntet, but Sydney is public testnet.

File: node/src/command.rs (hunk 1 above)
Comment (akorchyn) [reply node/src/command.rs:63]: +1,

**Review by akorchyn — CHANGES_REQUESTED** [review]

File: node/src/command.rs (hunk 1 above)
Comment (obsessed-cake) [reply node/src/command.rs:63]: I decided to store the configuration of chain specification in code for Sydney mainnet.
But I didn't know what network to use for it.

Comment (uchu) [reply node/src/command.rs:63]: @obsessed-cake There will be no Sydney-mainnet. Sydney will always remain one net. But at some moment we'll have Mainnet (without any Sydney name)

Comment (uchu) [reply node/src/command.rs:63]: We can just use "Mainnet" name  if you want to address the upcoming mainnet.

**Comment by uchu** [comment]
Of course we might need to find this code later, but I'm ok with removing it. At least we can always find it in history.
Up to @akorchyn to decide.

**Review by uchu — APPROVED** [review]

File: node/src/runtime/mainnet.rs [hunk 2]
```diff
@@ -23,22 +23,6 @@ impl ValidatorIdentity {
 			},
 		}
 	}
-
-	pub fn from_pub(ed: &str, sr: &str, _ecdsa: &str) -> ValidatorIdentity {
```
Comment (akorchyn) [inline node/src/runtime/mainnet.rs:27]: Let's keep it for future. it's easier to manipulate using this. At least for future.
#[allow(dead_code)]

Comment (obsessed-cake) [reply node/src/runtime/mainnet.rs:27]: Done.

**Pull request merged**